File: pkg/codegen/testing/utils/testdata/inline-invokes-pp/go/inline-invokes.go
package main

import (
	"github.com/pulumi/pulumi-aws/sdk/v5/go/aws/ec2"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
)

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		invokeLookupVpc, err := ec2.LookupVpc(ctx, &ec2.LookupVpcArgs{
			Default: pulumi.BoolRef(true),
		}, nil)
		if err != nil {
			return err
		}
		_, err = ec2.NewSecurityGroup(ctx, "webSecurityGroup", &ec2.SecurityGroupArgs{
			VpcId: invokeLookupVpc.Id,
		})
		if err != nil {
			return err
		}
		return nil
	})
}

File: tests/testdata/codegen/inline-invokes-pp/nodejs/inline-invokes.ts
import * as pulumi from "@pulumi/pulumi";
import * as aws from "@pulumi/aws";

const webSecurityGroup = new aws.ec2.SecurityGroup("webSecurityGroup", {vpcId: String(aws.ec2.getVpc({
    "default": true,
}).then(invoke => invoke.id))});
